service_discovery/proto/samples/v1/hello_world_service.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+// Hello World Service sample definition
+//
+// The protobuf definitions for the Hello World service sample
+
+syntax = "proto3";
+package hello_world;
+
+// The service entry point to the Hello World service. This simple service only has one method to
+// show the basic flow of applications calling one another
+service HelloWorld {
+    // Method which just logs and returns a message "Hello, {input string}" when it is called
```

Comment:
Please keep your indentation consistent.  You use 4 spaces here and 2 spaces later in the file.